### PR TITLE
Fix/offer button

### LINKED ISSUE #602: Add restriction when creating an offer, if the sponsor haven't finish his profile information

### PR DESCRIPTION
**Pull request by Edgar097**
Resolves #602 

### What does this PR do?
When clicking the add offer, if the profile is not complete, add a restriction until the data is enter

### How should this be tested?
- Make run
- Login as Sponsor
- Add an offer

### CODE REVIEW

File: webapp/src/components/Signup/Signup.js [hunk 1]
```diff
@@ -434,6 +434,13 @@ const Signup = ({ isHome, isModal, isSideBar }) => {
 
   }, [errorpreRegisterLifebank])
 
+  useEffect(()=>{
+    if(open){
+      handleSetField('email', ' ')
```
Comment (JustinCast) [inline webapp/src/components/Signup/Signup.js:439]: Why an empty string?.

File: webapp/src/language/en.json [hunk 2]
```diff
@@ -443,7 +443,8 @@
       "selectDateRange": "Select date range",
       "offerCreation": "Offer Created Successfully",
       "offerStateActive": "Active Offers",
-      "offerStateInactive": "Inactive Offers"
+      "offerStateInactive": "Inactive Offers",
+      "info": "In order for you to add an Offer, please finish your profile information"
```
Comment (JustinCast) [inline webapp/src/language/en.json:447]: please ~finish~ complete your profile information?

File: webapp/src/language/es.json [hunk 3]
```diff
@@ -442,7 +442,8 @@
       "selectDateRange": "Seleccione el rango de fecha",
       "offerCreation": "Oferta creada correctamente",
       "offerStateActive": "Ofertas Activas",
-      "offerStateInactive": "Ofertas Inactivas"
+      "offerStateInactive": "Ofertas Inactivas",
+      "info": "Para que pueda agregar una Oferta, complete la información de su perfil"
```
Comment (JustinCast) [inline webapp/src/language/es.json:446]: Why "Oferta" is in capital case?

File: webapp/src/routes/OffersManagement/OffersManagement.js [hunk 4]
```diff
@@ -385,7 +435,7 @@ const OffersManagement = () => {
         className={classes.fab}
         color="secondary"
         aria-label="add"
-        onClick={() => setOpenGenericFormAddVariant(true)}
+        onClick={() => checkAvailableFields()}
```
Comment (JustinCast) [inline webapp/src/routes/OffersManagement/OffersManagement.js:438]: `onClick={checkAvailableFields}`

**Review by JustinCast — CHANGES_REQUESTED** [review]

**Review by JustinCast — APPROVED** [review]

**Review by JustinCast — APPROVED** [review]

**Review by steph222 — APPROVED** [review]

File: webapp/src/routes/OffersManagement/OffersManagement.js [hunk 5]
```diff
@@ -127,6 +126,58 @@ const OffersManagement = () => {
...
+    if (!profile.benefit_description)
+      pendingFieldsObject = {
+        ...pendingFieldsObject,
+        benefit_description: false
+      }
+
+    if (!profile.location)
+      pendingFieldsObject = { ...pendingFieldsObject, location: false }
+
+    if(Object.keys(pendingFieldsObject).length>0){
```
Comment (leisterfrancisco) [inline webapp/src/routes/OffersManagement/OffersManagement.js:170]: Add space before and after the `>` symbol

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]
Looks great to me, but what if we change the box message color to yellow? to try to represent a kind of a warning alert?

**Review by leisterfrancisco — APPROVED** [review]

**Pull request merged**